train.py

```diff
@@ -189,13 +194,14 @@ def dev_step(x1_batch, x2_batch, y_batch):
                              siameseModel.input_y: y_batch,
                              siameseModel.dropout_keep_prob: FLAGS.dropout_keep_prob,
             }
-        step, loss, dist, summary = sess.run([global_step, siameseModel.loss, siameseModel.distance, summaries_merged],  feed_dict)
+        step, loss, dist, summary, out1, out2 = sess.run([global_step, siameseModel.loss, siameseModel.distance, summaries_merged,,siameseModel.out1,siameseModel.out2],  feed_dict)
         time_str = datetime.datetime.now().isoformat()
```

Comment:
out1,  out2 can be normal numpy arrays -- see tf.Session.run (https://www.tensorflow.org/api_docs/python/tf/Session#run)